packages/components/message-box/src/index.vue

```diff
@@ -260,6 +261,8 @@ export default defineComponent({
         : ''
     })
 
+    const btnSize = useSize()
```

Comment:
Sooooory, it should be
```suggestion
    const btnSize = useSize(
      computed(() => props.buttonSize),
      { prop: true, form: true, formItem: true }
    )
```

Reply:
ok